src/main/java/org/kuali/kfs/sys/document/web/renderers/GroupTitleLineRenderer.java

```diff
@@ -424,12 +427,16 @@ protected void renderUploadCell(PageContext pageContext, Tag parentTag) throws J
 
                 cancelButtonTag.setPageContext(pageContext);
                 cancelButtonTag.setParent(parentTag);
-                cancelButtonTag.setAlt("Cancel import of " + accountingLineGroupDefinition.getGroupLabel() + " accounting lines");
-                cancelButtonTag.setTitle("Cancel import of " + accountingLineGroupDefinition.getGroupLabel() + " accounting lines");
-                cancelButtonTag.setOnclick(getHideImportName() + "(\\\'" + showLink + "\\\',\\\'" + uploadDiv + "\\\');return false;");
+                cancelButtonTag.setAlt(
+                        "Cancel import of " + accountingLineGroupDefinition.getGroupLabel() + " accounting lines");
+                cancelButtonTag.setTitle(
+                        "Cancel import of " + accountingLineGroupDefinition.getGroupLabel() + " accounting lines");
+                cancelButtonTag.setOnclick(
+                        getHideImportName() + "(\\\'" + showLink + "\\\',\\\'" + uploadDiv + "\\\');return false;");
                 cancelButtonTag.doStartTag();
                 cancelButtonTag.doEndTag();
 
+

```

Comment:
I don't think this extra space is needed.

Reply:
removed